src/server/dataSource.js

```diff
@@ -61,7 +66,7 @@ fluid.defaults("sjrk.storyTelling.server.dataSource.couch.view", {
 
 // A CouchDB DataSource for a single story, for use on the storyView and storyEdit pages
 fluid.defaults("sjrk.storyTelling.server.dataSource.couch.story", {
-    gradeNames: ["sjrk.storyTelling.server.dataSource.couch.core", "kettle.dataSource.CouchDB"],
+    gradeNames: ["fluid.modelComponent", "sjrk.storyTelling.server.dataSource.couch.core", "kettle.dataSource.CouchDB"],
```

Comment:
Doesn't seem to be any need for this to be a modelComponent

Reply:
This was leftover from a previous attempt to communicate the schemaVersion to the data record. Removing it!